src/pages/home/report/ReportTypingIndicator.js

```diff
@@ -74,16 +73,13 @@ class ReportTypingIndicator extends React.Component {
 
             default:
                 return (
-                    <Text
-                        style={[
-                            styles.chatItemComposeSecondaryRowSubText,
-                            styles.chatItemComposeSecondaryRowOffset,
-                        ]}
-                        numberOfLines={1}
-                    >
-                        {this.props.translate('reportTypingIndicator.multipleUsers')}
-                        {` ${this.props.translate('reportTypingIndicator.areTyping')}`}
-                    </Text>
+                    <TextWithEllipsis
+                        leadingText={this.props.translate('reportTypingIndicator.multipleUsers')}
+                        trailingText={` ${this.props.translate('reportTypingIndicator.areTyping')}`}
+                        textStyle={[styles.chatItemComposeSecondaryRowSubText]}
+                        wrapperStyle={[styles.chatItemComposeSecondaryRow]}
+                        leadingTextParentStyle={styles.chatItemComposeSecondaryRowOffset}
+                    />
```

Comment:
Looking at the code of this file, it seems it can be optimized. 

`leadingText` and `trailingText` can be moved to variables and then used in the final `TextWithEllipsis` element.